macros/HTMLRef.ejs

```diff
@@ -27,6 +27,20 @@ switch (env.locale) {
     default: break;
 }
 
+var text = mdn.localStringMap({
+    'en-US': {
+        'Input_types': '<code>&lt;input&gt;</code> types'
+    }
+});
+
+var currentSection = $0;
```

Comment:
You don't actually need `currentSection` and the `state` function. Remove both.

Reply:
No? Is there another mechanism for allowing setting which section is open by default? Having this lets me pass in "Input_types" into HTMLRef to get that section open by default when looking at <input> and its subpages.

Reply:
I would say, either use `<% if (env.slug.includes("/HTML/Element/input")) { %>` or the `state` function. Not both.